t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
+
+test_expect_success 'custom merge backend with delay' '
+	git reset --hard anchor &&
+	echo "* merge=union" >.gitattributes &&
+	echo "text merge=custom" >>.gitattributes &&
+
+	git config --replace-all \
+	merge.union.driver "true" &&
+	git config --replace-all \
+	merge.union.name "always ours" &&
+
+	git config --replace-all \
+	merge.custom.driver "./custom-merge %O %A %B 0 %P" &&
+	git config --replace-all \
+	merge.custom.name "custom merge driver for testing" &&
```

Comment:
The name is not really needed, is it? Let's keep the test short and sweet; the name "custom" of the custom driver (or "sleep-one-second", as I would prefer) is self-explanatory enough.